products/java.md

```diff
@@ -26,13 +26,15 @@ releases:
     eol: 2023-03-21
     latest: "19.0.2"
     releaseDate: 2022-09-20
+    link: https://www.oracle.com/java/technologies/javase/19-0-2-relnotes.html
```

Comment:
do we need this ? because changelogtemplate have to work with current share 
do you get any problem on this version's link (without your patch ) ?